x-pack/plugins/security_solution/public/common/lib/cell_actions/helpers.ts

```diff
@@ -35,6 +35,7 @@ export const COLUMNS_WITH_LINKS = [
   {
     columnId: SIGNAL_RULE_NAME_FIELD_NAME,
     label: i18n.VIEW_RULE_DETAILS,
+    linkField: 'signal.rule.id',
```

Comment:
changing this to `kibana.alert.rule.uuid` fixes it for the 8.0 branch. For 7.16 it probably has to stay as `signal.rule.id`